src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.scss

```diff
@@ -11,10 +11,15 @@
 
   .euiSelectableListItem__content {
     cursor: default;
+    padding-left: 0;
+    margin-right: 0;
 
 
   .euiSelectableListItem__text {
-    max-height: 100%; 
+    max-height: 100%;
+  }
+  .euiSelectableListItem__icon, .euiSelectableListItem__prepend {
```

Comment:
Should be an extra line between  .euiSelectableListItem__icon, and .euiSelectableListItem__text